SPECS/dracut/20overlayfs/overlayfs-mount.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/sh
```

Comment:
would it make sense to add a comment at the top including:
- what kernel parameters drive this module and their meaning
- what setup we get out of it